frontend/src/utils/index.js

```diff
@@ -160,12 +160,15 @@ export function routes (router, includeRoutesWithProjectScope) {
   return filter(defaultRoute.children, hasMenu)
 }
 
-export function namespacedRoute (route, namespace) {
-  const name = routeName(route)
+export function namespacedRoute (route, namespace, name = undefined) {
```

Comment:
where do we use the name parameter

Reply:
not used (yet), slipped in from webterm branch. I have removed it